packages/ember-routing/tests/system/dsl_test.js

```diff
@@ -18,7 +18,6 @@ QUnit.module('Ember Router DSL', {
 });
 
 QUnit.test('should fail when using a reserved route name', function() {
-  expectDeprecation('this.resource() is deprecated. Use this.route(\'name\', { resetNamespace: true }, function () {}) instead.');
   let reservedNames = ['array', 'basic', 'object', 'application'];
 
   expect((reservedNames.length * 2) + 1);
```

Comment:
Need to tweak this somewhat, the CI failure is due to it thinking it has 9 assertions and only actually getting 5